### PR TITLE
Pkg/separate deps

### PR DESCRIPTION
**Pull request by kalikaneko**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -129,7 +133,7 @@ def get_versions(default={}, verbose=False):
     packages=find_packages('src', exclude=['leap.keymanager.tests']),
     package_dir={'': 'src'},
     test_suite='leap.keymanager.tests',
-    install_requires=utils.parse_requirements(),
```
Comment (ivanalejandro0) [inline setup.py]: you could use `utils.parse_requirements(reqfiles=["requirements.pip", "pkg/requirements-leap.pip"])`

Comment (ivanalejandro0) [inline setup.py]: here or at the top, to prevent calling that code twice.

Comment (kalikaneko) [inline setup.py]: yum, I don't know even my own code :P

Comment (kalikaneko) [inline setup.py]: hmm no, it doesn't work this way. the function get_reqs from files shortcircuits as soon as one of the alternatives is found in the filesystem. 
I'll update the docstring

**Pull request merged**